package.json

```diff
@@ -14,13 +14,15 @@
     "eslint": "8.15.0",
     "eslint-config-kinvey-platform": "0.1.6",
     "eslint-plugin-import": "2.26.0",
-    "mocha": "10.0.0",
+    "jscpd": "3.4.5",
+    "mocha": "9.2.2",
```

Comment:
why the revert?

Reply:
There was an error in the CI - it seems that the node version there is < 12.2 - with 12.21 locally the tests passed, but with 12.15.0 they did not.The 9.2.2. mocha version has the needed security fixes and causes no problems.